utils/vim/syntax/swift.vim

```diff
@@ -141,6 +139,13 @@ syn match swiftTypeDeclaration skipwhite skipempty nextgroup=swiftType,swiftInOu
 syn match swiftTypeDeclaration skipwhite skipempty nextgroup=swiftType
       \ /->/
 
+syn match swiftKeyword
+      \ /\<case\>/
+syn region swiftCaseLabelRegion
+      \ matchgroup=swiftKeyword start=/\<case\>/ matchgroup=Delimiter end=/:/ oneline contains=TOP
```

Comment:
I would not like to add `oneline` but there is a concern about the `case` label in the enum-declaration. I think that most cases will be saved even with `oneline`.